sensor_msgs/Image img
---
float64 treasure_x
float64 treasure_y
